web/src/i18n/i18n.js

```diff
@@ -17,7 +17,7 @@ function loadLocaleMessages() {
 }
 
 export default new VueI18n({
-  locale: process.env.VUE_APP_I18N_LOCALE || 'en',
+  locale: navigator.language.split('-')[0] || 'en',
```

Comment:
From the link you shared, perhaps this is more complete:

```suggestion
  locale: navigator.language.split('-')[0] || process.env.VUE_APP_I18N_LOCALE || 'en',
```